backend/data_tools/data/agreements_and_blin_data.json5

```diff
@@ -323,6 +323,9 @@
       proc_shop_fee_percentage: 0,
       services_component_id: 1,
       clin_id: 1,
+      on_hold: false,
+      certified: false,
+      closed: false,
```

Comment:
🤔 wonder what would it mean for a budget line to be on-hold, certified, and closed?

Reply:
I know I've heard sheila casually mention that business-process wise, they may want to put a BL "on hold" but I don't know what implications that has and we've not really built anything into OPS's functionality for it yet.  Closed means they've spent/invoiced out all the money on an obligated BL and it's done for.  not 100% sure on certified.